src/main/java/dao/CsvDataParserImpl.java

```diff
@@ -0,0 +1,24 @@
+package dao;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+import model.Operation;
+
+public class CsvDataParserImpl implements CsvDataParser {
+    private static final int ROW_POSITION_AT_ZERO = 0;
+    private static final int ROW_POSITION_AT_ONE = 1;
+    private static final int ROW_POSITION_AT_TWO = 2;
```

Comment:
uninformative naming. change to OPERATION_INDEX / OPERATION_POSITION
we should name after role, not value. The format can change later on  - suppose fruit will be first, operation second. It would be a little confusing to make changes to it